components/developer/clang-40/Makefile

```diff
@@ -47,15 +49,25 @@ CMAKE_PREFIX= /usr/clang/$(COMPONENT_MAJOR_VERSION)
 
 PATH=$(PATH.illumos)
 
+PKG_MACROS+= GCC_VERSION=$(GCC_VERSION)
+
 COMPONENT_POST_UNPACK_ACTION += \
   $(UNPACK) $(USERLAND_ARCHIVES)$(COMPONENT_ARCHIVE_1) && \
 	mv $(CLANG_NAME)-$(COMPONENT_VERSION).src $(COMPONENT_SRC)/tools/clang
 
-COMPONENT_PRE_CONFIGURE_ACTION =        (cp -a $(SOURCE_DIR)/* $(@D))
+COMPONENT_PREP_ACTION = ( \
```

Comment:
Can we instead use compile-time make variables? This would be a bit prettier. 
However, if it's hard, I'm fine with this.